migrations/2_dev_dependencies.js

```diff
@@ -1,10 +0,0 @@
-const migrateDependencies = require("../src/migration/dependencies")
-
-module.exports = function(deployer, network, accounts) {
-  return migrateDependencies({
-    artifacts,
-    deployer,
-    network,
-    account: accounts[0],
-  })
-}
```

Comment:
Is this file really no longer needed? Will this change our current dev-environemnt?

Reply:
Felix suggested migrating all dependencies within src/migration/PoC_dfusion.js  and src/migration/snapp_auction.js
I basically copy-pasted the code into there...

Reply:
Cool beans! Just wondering